Header header

uint8   utc_hour
uint8   utc_minute
uint16  utc_millisec
float64 latitude
float64 longitude
uint8   fix_quality
uint8   n_sv_used
float32 HDOP
float64 orthometric_height
float32 geoid_undulation
float32 differential_age
uint16  ref_station_ID